win32ss/gdi/ntgdi/freetype.c

```diff
@@ -5428,7 +5431,8 @@ NtGdiGetFontFamilyInfo(HDC Dc,
     }
 
     /* Allocate space for a safe copy */
-    Info = ExAllocatePoolWithTag(PagedPool, Size * sizeof(FONTFAMILYINFO), GDITAG_TEXT);
+    DataSize = InfoCount * sizeof(FONTFAMILYINFO);
```

Comment:
Might want to add a check for overflow while you're at it. This should use `RtlULongMult` (or at least verify that `DataSize > InfoCount`)

Reply:
OK. Used `RtlULongMult`.